data_steward/cdr_cleaner/clean_cdr.py

```diff
@@ -140,6 +140,14 @@
     (StoreNewPidRidMappings,),
     (TruncateRdrData,),
     (RemoveParticipantsUnder18Years,),
+    # execute map_questions_answers_to_omop before PpiBranching gets executed
```

Comment:
@nishanthpp93 This is good, but we also should add the dependency in the PPI_branching rule dependency list.

Reply:
I see, thanks. Will add it in the cleaning rule class.

Reply:
@ksdkalluri I am looking at the BaseCleaningRule class and it looks like depends_on can raise an error because this cleaning rule is not base classed. Let me know if that's okay.

Reply:
Just tested this and it looks like the PpiBranching class cannot be instantiated with it. It runs into the same error.

Reply:
I can instead add a comment there about the dependency along with a TODO. Let me know your thoughts.

Reply:
okay, that's a bummer.  Just add a TODO item about the dependency and it will be good for now.